### PR TITLE
feat: add gtag events and pageview, improve seo

### PR DESCRIPTION
**Pull request by ramonxm**
## Description

Adicionando eventos do gtag e adicionando meta description por default

## Developer Checks

- [ ] PR title & commits adhere to [Conventional Commit Specification](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] PR is targeting correct branch, is up-to-date, & no merge conflicts
- [ ] Tested on browser device
- [ ] Tested on Mobile device

### CODE REVIEW

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.47 | ✅ |
| First Contentful Paint | 1.8 s |  |
| Time to Interactive | 5.8 s |  |
| Speed Index | 2.8 s |  |
| Total Blocking Time | 1,930 ms |  |
| Largest Contentful Paint | 6.4 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.92 | ✅ |
| First Contentful Paint | 2.0 s |  |
| Time to Interactive | 6.3 s |  |
| Speed Index | 2.4 s |  |
| Total Blocking Time | 140 ms |  |
| Largest Contentful Paint | 2.1 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -22,6 +36,10 @@ function App({ Component, pageProps }: AppProps) {
         <meta name="apple-mobile-web-app-status-bar-style" content="#81CAA8" />
         <meta name="msapplication-navbutton-color" content="#81CAA8" />
         <link rel="manifest" href="/manifest.json" />
+        <meta
```
Comment (ramonxm) [inline src/pages/_app.tsx:39]: Sinceramente, não entendi ainda o motivo do Google não estar ranqueando essa tag. Mas seguindo o que fizeram utilizando o Next Seo, ainda mantiveram essa tag no app. Então, veremos;

Comment (jonataspinto) [reply src/pages/_app.tsx:39]: Pode mandar o exemplo?

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,14 +1,28 @@
...
 import SEO from '../../next-seo.config';
 import NextNProgress from 'nextjs-progressbar';
+import { useRouter } from 'next/router';
 
 function App({ Component, pageProps }: AppProps) {
+  const router = useRouter();
+
+  useEffect(() => {
+    const handleRouteChange = (url: string) => {
+      gtag.pageview(url);
```
Comment (jonataspinto) [inline src/pages/_app.tsx:18]: Não entendi a motivação? acho que ja está rolando.

File: src/pages/_app.tsx (hunk 1 above)
Comment (ramonxm) [reply src/pages/_app.tsx:39]: https://github.com/willianjusten/willianjusten.com.br

Fui dar uma olhada no site do willian que tem um ótimo seo, e mesmo ele utilizando next seo ele passa essa description

File: src/pages/_app.tsx (hunk 2 above)
Comment (ramonxm) [reply src/pages/_app.tsx:18]: consegue verificar pra mim no painel do google analytics? ao meu ver não tinha setado pageview só com aquilo que eu tinha feito, mas se já estiver então done. nesse caso ai é por conta da rota de perguntas frequentes.

File: src/pages/_app.tsx (hunk 1 above)
Comment (jonataspinto) [reply src/pages/_app.tsx:39]: Entendi, então vamos mergear isso!

File: src/pages/_app.tsx (hunk 2 above)
Comment (jonataspinto) [reply src/pages/_app.tsx:18]: image

Comment (jonataspinto) [reply src/pages/_app.tsx:18]: ta rolando ja

Comment (ramonxm) [reply src/pages/_app.tsx:18]: fecho ent, subir o fix

**Comment by jonataspinto** [comment]
<details>
    <summary>Lighthouse result for https://www.estartandodevs.com.br/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.77 | ✅ |
| First Contentful Paint | 1.2 s |  |
| Time to Interactive | 5.0 s |  |
| Speed Index | 1.5 s |  |
| Total Blocking Time | 940 ms |  |
| Largest Contentful Paint | 1.9 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>
<details>
    <summary>Lighthouse result for http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/</summary>
    

  | Metric | Score |  |
  | --- | --- | --- |
  | Performance | 0.93 | ✅ |
| First Contentful Paint | 1.8 s |  |
| Time to Interactive | 6.1 s |  |
| Speed Index | 2.1 s |  |
| Total Blocking Time | 120 ms |  |
| Largest Contentful Paint | 1.8 s |  |
| Cumulative Layout Shift | 0.001 |  |

    
</details>

**Review by jonataspinto — COMMENTED** [review]
| Device | URL |
|--|--|
| desktop | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

     
| Device | URL |
|--|--|
| mobile | http://estartandodevs-site.s3-website-sa-east-1.amazonaws.com/ |

Not what you expected? Are your scores flaky? [Run Lighthouse on Foo](https://www.foo.software/lighthouse)
If scores continue to be inconsistent consider [running all audits on Foo](https://github.com/foo-software/lighthouse-check-action#usage-foos-automated-lighthouse-check-api)

**Review by jonataspinto — APPROVED** [review]

**Pull request merged**